src/resolvers/query/card/index.js

```diff
@@ -10,17 +10,17 @@ export default async function card(parent, args) {
   const response = await getStripeCard(stripeCustomerId);
 
   const customerInfo = response;
-  const cardInfo = response.sources.data[0] || {};
+  const cardInfo = response.sources.data[0] || null;
```

Comment:
```suggestion
  const cardInfo = response.sources.data[0];
```
Purely a nitpick style thing but I think we can omit the `|| null`, this will make `cardInfo` `undefined` instead. Both are falsey, so shouldn't matter.